modules/nf-scil/reconst/diffusivitypriors/main.nf

```diff
@@ -0,0 +1,75 @@
+
+
+process RECONST_DIFFUSIVITYPRIORS {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+        tuple val(meta), path(fa), path(ad), path(md), path(priors)
+
+    output:
+        tuple val(meta), path("*para_diff.txt")     , emit: para_diff, optional: true
+        tuple val(meta), path("*iso_diff.txt")      , emit: iso_diff, optional: true
```

Comment:
I may be mistaken, but this pattern ``*para_diff.txt`` or ``*iso_diff.txt`` can also match ``mean_para_diff.txt`` and ``mean_iso_diff.txt`` (and the all_* files), no? I would do something as suggested just to be sure all output channel are the expected shape, but feel free to change it to something else if you prefer! 
```suggestion
        tuple val(meta), path("*__iso_diff.txt")      , emit: iso_diff, optional: true
        tuple val(meta), path("all_iso_diff.txt")      , emit: all_iso_diff, optional: true
```

Reply:
You don't need the all_*.txt output, but I did change the output format for the mean differentiators